docs/rest-api/rest-notebook.md

```diff
@@ -93,7 +93,21 @@ limitations under the License.
     </tr>
     <tr>
       <td> sample JSON input </td>
-      <td><pre>{"name": "name of new notebook"}</pre></td>
+      <td><pre>
+{
+  "name": "name of new notebook", 
+  "paragraphs": [
```

Comment:
Fine. I think adding description of parameters requires changing of all REST APIs, and we can file a new issue to JIRA.
I'll just add a new sample which doesn't have paragraphs.
Thanks for quick review!